### PR TITLE
[feature/OPT-796] to dev

### PR DESCRIPTION
**Pull request by dantolin-iriusrisk**

### CODE REVIEW

File: slp_tfplan/tests/unit/transformers/dataflow/strategies/test_dataflow_creation_strategy.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        target_component.name = target_name
+
+        # AND a bidirectional flag
+
+        # WHEN create_dataflow is invoked
+        dataflow = create_dataflow(source_component, target_component, bidirectional)
+
+        # THEN a dataflow is created with the right ID
+        if bidirectional:
+            assert dataflow.id == deterministic_uuid(hash(source_id) + hash(target_id))
```
Comment (jmgarcia-iriusrisk) [inline slp_tfplan/tests/unit/transformers/dataflow/strategies/test_dataflow_creation_strategy.py:56]: may it be better set the id value as static in the test?
So we could get an error if the function return changes

File: slp_tfplan/tests/unit/transformers/dataflow/strategies/test_dataflow_creation_strategy.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        else:
+            assert dataflow.id == deterministic_uuid(f'{source_id}-{target_id}')
+
+        # AND its name is right
+        assert dataflow.name == f'{source_name} to {target_name}'
```
Comment (jmgarcia-iriusrisk) [inline slp_tfplan/tests/unit/transformers/dataflow/strategies/test_dataflow_creation_strategy.py:61]: Should the name change if Bidirectional?

Comment (dantolin-iriusrisk) [reply slp_tfplan/tests/unit/transformers/dataflow/strategies/test_dataflow_creation_strategy.py:61]: Yes, I think so. Maybe `f'{source_name} to {target_name} (bidirectional)'` to not overcomplicate things?

File: slp_tfplan/tests/unit/transformers/dataflow/strategies/test_dataflow_creation_strategy.py (hunk 1 above)
Comment (dantolin-iriusrisk) [reply slp_tfplan/tests/unit/transformers/dataflow/strategies/test_dataflow_creation_strategy.py:56]: Agree.

**Review by jmgarcia-iriusrisk — APPROVED** [review]

**Pull request merged**